pgrx-pg-sys/build.rs

```diff
@@ -530,9 +530,9 @@ fn impl_pg_node(
 
         // impl Rust's Display trait for all nodes
         pgnode_impls.extend(quote! {
-            impl std::fmt::Display for #struct_name {
-                fn fmt(&self, f: &mut std::fmt::Formatter<'_>) -> std::fmt::Result {
-                    write!(f, "{}", self.display_node() )
+            impl ::core::fmt::Display for #struct_name {
+                fn fmt(&self, f: &mut ::core::fmt::Formatter<'_>) -> ::core::fmt::Result {
+                    f.write_str(&self.display_node())
```

Comment:
Since we already have a string, we should just use `self.display_node().fmt(f)`. This will respect padding and alignment parameters. (It won't be any faster though).

Reply:
oh, that sounds good.